src/App.js

```diff
@@ -1,53 +1,97 @@
 import React, { useState, useEffect } from "react";
 import AddTodoForm from "./AddTodoForm";
-import TodoList from "./TodoList";
+
+// Airtable API details
+const API_BASE = "https://api.airtable.com/v0/appG0eZm9k8QxyX4G/Default"; // Correct Base ID
+const HEADERS = {
+  Authorization: `Bearer patgzRbxdSabSy0Xr.72ba65f33002e147496cb710c86a48c77dbfe3fc5f0e630aae3dfc3182a676d5`, // Correct token format
+  "Content-Type": "application/json",
+};
```

Comment:
This is where you should use the keys from the env. you should ever use keys directly or else the keys can be stolen and your app compromised

```suggestion
const API_BASE = `https://api.airtable.com/v0/${process.env.REACT_APP_AIRTABLE_BASE_ID}/${process.env.REACT_APP_AIRTABLE_TABLE_NAME}`; // Correct Base ID
const HEADERS = {
  Authorization: `Bearer ${process.env.REACT_APP_AIRTABLE_API_TOKEN}`, // Correct token format
  "Content-Type": "application/json",
};
```